browser/ui/views/frame/brave_browser_frame_mac.mm

```diff
@@ -56,3 +57,20 @@
   result->new_toggle_state = !model->empty() && model->GetActiveWebContents() &&
                              model->GetActiveWebContents()->IsAudioMuted();
 }
+
+bool BraveBrowserFrameMac::ExecuteCommand(
+    int32_t command,
+    WindowOpenDisposition window_open_disposition,
+    bool is_before_first_responder) {
+  // Ignoring the ctrl+w command when the active tab is pinned
```

Comment:
nit: Could you leave more about is_before_first_reponder? I think some readers might wonder how this is related.

Reply:
Could you move this comment to inside the if() condition?

Reply:
Done!